src/static/static/img/cc-pdm_icon.svg

```diff
@@ -0,0 +1,5 @@
+<svg viewBox="0 0 30 30" xmlns="http://www.w3.org/2000/svg">
+	<g id="icon" fill="currentColor">
+		<path d="M15 0C6.716 0 0 6.716 0 15c0 8.283 6.716 15 15 15 8.284 0 15-6.717 15-15 0-8.284-6.716-15-15-15zm0 27.187C8.28 27.187 2.812 21.72 2.812 15c0-1.407.243-2.758.683-4.015l4.767 2.122a10.081 10.081 0 00-.198 1.991c0 5.988 4.47 7.675 7.282 7.675a7.314 7.314 0 004.217-1.32c.214-.153.407-.307.585-.464l-2.187-2.657a2.496 2.496 0 01-.2.206c-.85.806-1.785.806-1.994.806-2.288 0-3.246-2.414-3.258-4.348l13.137 5.85c.015.002.028.002.041.005-2.073 3.774-6.086 6.336-10.687 6.336zm11.745-9.073l-13.55-6.048c.489-.885 1.267-1.55 2.431-1.55.675 0 1.21.223 1.632.506.175.126.327.252.456.38l2.442-2.514c-1.64-1.41-3.505-1.663-4.725-1.663-2.866 0-4.895 1.24-6.09 3.121L4.81 8.326C6.99 5.007 10.743 2.811 15 2.811c6.72 0 12.188 5.467 12.188 12.188 0 1.088-.145 2.14-.414 3.145-.011-.01-.018-.02-.03-.03z" />
+	</g>
+</svg>
```

Comment:
A new line here would be 👍

Reply:
Done in https://github.com/WordPress/openverse-frontend/pull/990/commits/2c5b05e65746a414427976f519ec61e1f12495ca